index.js

```diff
@@ -110,23 +113,21 @@ function reduceRules(ignore, opts) {
 	};
 }
 
-
 function api(stylesheet, ignore, opts) {
+	opts = defaults(opts || {}, _default);
 
-	opts = _.defaults(opts || {}, _default);
-
-	if (!_.isArray(ignore)) {
+	if (!Array.isArray(ignore)) {
 		ignore = [ignore];
 	}
 
-	var sheet;
+	let sheet;
 	try {
 		sheet = css.parse(read(stylesheet));
-	} catch (err) {
+	} catch (error) {
 		sheet = css.parse(stylesheet);
 	}
 
-	sheet.stylesheet.rules = _.reduce(sheet.stylesheet.rules, reduceRules(ignore, opts), []);
+	sheet.stylesheet.rules = (sheet.stylesheet.rules || []).reduce(reduceRules(ignore, opts), []);
```

Comment:
If this can't be undefined let me know and I'll remove the fallback

Reply:
According to the [readme](https://github.com/reworkcss/css#stylesheet) and the [code](https://github.com/reworkcss/css/blob/master/lib/parse/index.js) this should always be an array.

If `css.parse(...)` fails it should throw an error and we won't get here so i think we could safely remove the fallback